deploy-manage/deploy/self-managed/install-elasticsearch-docker-configure.md

```diff
@@ -63,15 +63,36 @@ To encrypt your secure settings with a password and have them persist outside th
 
 For example:
 
+::::{tab-set}
+:group: docker
+:::{tab-item} Latest
+:sync: latest
 ```sh subs=true
 docker run -it --rm \
 -v full_path_to/config:/usr/share/elasticsearch/config \
 docker.elastic.co/elasticsearch/elasticsearch:{{version.stack}} \
 bin/elasticsearch-keystore create -p
 ```
+:::
+
+:::{tab-item} Specific version
+:sync: specific
+Replace `<SPECIFIC.VERSION.NUMBER>` with the {{es}} version number you want. For example, you can replace `<SPECIFIC.VERSION.NUMBER>` with {{version.stack.base}}.
```

Comment:
thoughout this page this is also not a "want" but instead should be the version of your docker image

Reply:
Nice one! Thanks for catching that. Fixed in a new commit